recipes/cccl-python/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "cccl-python" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cccl/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3ded63682f94071685cd6492cee70ad2a1feae02abfb3fbcce50e2d2f22edf4e
+
+build:
+  script: |
+    cd python/cuda_cccl
+    {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Because of the way the CCCL C lib is built we also need the CUDA compiler here
```suggestion
    - {{ compiler('cxx') }}
    - {{ compiler('cuda') }}
```